packages/turf-transform-translate/test.js

```diff
@@ -0,0 +1,58 @@
+const test = require('tape');
+const path = require('path');
+const fs = require('fs');
+const load = require('load-json-file');
+const write = require('write-json-file');
+const helpers = require('@turf/helpers');
```

Comment:
Since the tests are now written in ES6 (NodeJS 6+) you can use the following when importing multiple methods from a module:

```javascript
const {point, featureCollection} = require('@turf/helpers');
```